common/src/main/java/org/apache/uniffle/common/util/RssUtils.java

```diff
@@ -274,6 +274,14 @@ public static String generatePartitionKey(String appId, Integer shuffleId, Integ
         Constants.KEY_SPLIT_CHAR, appId, String.valueOf(shuffleId), String.valueOf(partition));
   }
 
+  public static <T> T loadExtension(Class<T> extCls, String clsPackage, Object obj) {
+    List<T> exts = loadExtensions(extCls, Arrays.asList(clsPackage), obj);
+    if (exts != null && exts.size() == 1) {
+      return exts.get(0);
+    }
+    return null;
```

Comment:
What will it happen if we have two extensions?

Reply:
Currently, this will not be illegal

Reply:
So maybe we should throw an IllegalException instead of returning null value.

Reply:
done